### ISSUE TITLE
 Local storage being used for uploads from sdcard 

### ISSUE THREAD

**Issue by tobiasKaminsky** [nextcloud/android#351]
If you upload say 10 files from SD card the android app will copy the 10 files to the local storage on the device before uploading.

Files can be found in /nextcloud/user@//...

This is extremely inefficient and wastes storage space on local device. This part of the upload process needs to be re-coded.

This only applies to files uploaded manually, auto upload does not do this.

https://help.nextcloud.com/t/local-storage-being-used-for-uploads-from-sdcard/3102

**Comment by iskradelta** [comment]
Maybe a reason why this was done in the first place, was to upload the file even if it has been deleted, moved or renamed after the upload has started,  especially if upload is chunked or wifi is lost meanwhile, is this the desired behavior?

**Comment by tobiasKaminsky** [comment]
Is this still happening?
@mario as you have a device with an external sdcard?

**Comment by tobiasKaminsky** [comment]
Verified, not happening anylonger.

**Comment by ankitsejwal** [comment]
Hi @tobiasKaminsky, I would like to report that this issue still persists. I am new to NextCloud and I uploaded around 35GB of files to NextCloud (External drive) on my Openmediavault using NextCloud Android app. I faced two issues:
Android phone storage got full (issue #2532) and
OMV disk space was full as well (see attached screenshot)

I was able to empty the phone through clearing data from NextCloud folder using a file manager. 
> However, I can't find NextCloud directory in my Openmediavault, can you please help me find the NextCloud directory as my sd card storage is now full and the uploads have stopped.


The scenario in short:
Uploading photos to NextCloud using Android app, on a NextCloud attached with External storage

What's happening:
Files are being uploaded to External drive :) but,
Phone and SD card on my Raspberry Pi (running OMV) ran out of storage :(

>Ideal scenario:
Upload photos to External storage attached to NextCloud without building unnecessary data onto phone or OMV SD card.

Thanks

![Screen Shot 2019-05-23 at 4 48 45 am](https://user-images.githubusercontent.com/13817482/58205322-9e9d1c00-7d21-11e9-93d1-9ff7eaddcd05.png)
